tests/general/URLBearTest.py

```diff
@@ -53,6 +63,42 @@ def test_precentage_encoded_url(self):
                                'yes-green.svg/200'),
                               LINK_CONTEXT.no_context])
 
+    def test_detect_enclosed_parenthesis_url_result(self):
+        valid_file = """
+        http://wik.org/Hello_(Adele_song)/200
+        """.splitlines()
+
+        result = get_results(self.uut, valid_file)
+        self.assertEqual(result[0].contents,
+                         [2, 'http://wik.org/Hello_(Adele_song)/200',
+                          LINK_CONTEXT.no_context])
+
+    def test_detect_trailing_char_url_result(self):
+        valid_file = """
+        http://google.com/trailing.
+        """.splitlines()
+
+        result = get_results(self.uut, valid_file)
+        self.assertEqual(result[0].contents,
+                         [2, 'http://google.com/trailing',
+                          LINK_CONTEXT.no_context])
+
+    def test_detect_example_url_result(self):
+        invalid_file = """
+        http://example.com
+        """.splitlines()
```

Comment:
Why is `example.com` an invalid url?

Reply:
The existing code for URLBear allows the user to input regex that specifies certain links to ignore, and [the run() function of URLBear specifies `example.com` as a link to ignore](https://github.com/coala/coala-bears/blob/2888639aa2427ae77ab74486e8750770a1e219c1/bears/general/URLBear.py#L135), hence why I marked `example.com` as invalid.